src/initialize_dae.jl

```diff
@@ -65,7 +65,7 @@ function _initialize_dae!(integrator, prob::DAEProblem,
             ShampineCollocationInit(), x)
   else
     _initialize_dae!(integrator, prob,
-            BrownFullBasicInit(), x)
+            BrownFullBasicInit(integrator.opts.abstol ), x)
```

Comment:
and reltol?

Reply:
should we pick the minimum? The function implementation for the check is 
```
integrator.opts.internalnorm(tmp,t) <= alg.abstol && return
```

Reply:
ahh yeah, that check only makes sense for abstol.